docs/Administration.rst

```diff
@@ -2456,18 +2460,45 @@ should be sent to you.
 Administrating spaces 
 ======================
 
-It is possible to restrict which users can create and/or manage spaces
+It is possible to restrict which users can create and/or manage spaces and templates
 through an easy to use interface .
 
 |image305|
 
-.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and only administrators (i.e.
-          the members of the group ``*:/platform/administrators``) are allowed to manage spaces.
+.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and templates and only administrators (i.e.
+          the members of the group ``*:/platform/administrators``) are allowed to manage spaces and templates.
           
 The spaces administration interface is accessible via the menu Administration --> Spaces.
 
 |image306|
 
+This Spaces menu contains two choices:
+
+- "Manage Templates" : The administrator can create, update, delete or even change permissions of some space templates.
```

Comment:
can we really do all of this already ?

Reply:
Modification are done in the documentation. Can you check it please?